src/components/Cart/Cart.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Divider from '@mui/material/Divider';
+import ProductCard from '../Product/ProductCard';
+import SharedGrid from '../shared/Grid/SharedGrid';
+import { UI_TEXT } from '../../constants/text';
+import {
+  CartContainer,
+  CartTitle,
+  CartRow,
+  EmptyCartText,
+} from './Cart.styles';
+import PurchaseButton from '../shared/Button/purchaseButton/PurchaseButton.jsx';
+
+const Cart = ({
+  items,
+  total,
+  onContinue,
+  isLoading = false,
+  isError = false,
+  isLoggedIn = true,
+}) => {
+  const validItems = Array.isArray(items) ? items : [];
+  const isCartEmpty = validItems.length === 0;
+  const canPurchase = isLoggedIn && !isCartEmpty && total > 0;
+
```

Comment:
Is it possible to see the cart without being logged in? 
Because if not it make no sense to check for logged in user as well.